### ISSUE TITLE
Better indicate external link behavior

### ISSUE THREAD

**Issue by aviannamiller** [omeka/Omeka#1108]
As with https://github.com/omeka/omeka-s/issues/2531:
> Links that open in new tabs are currently indicated by external link icons. It is best practice to indicate if a link opens in a new tab, so we should be supplying an additional label accessible to screen readers ("Opens in new tab"). A tooltip that reads the same would also be appropriate, ultimately.

This will mainly affect "View Public Page" links, but there may be additional links to adjust.

**Comment by kimisgold** [comment]
I have a treatment on the `a11y/label-external-links` branch. The markup looks like this: 

`<a href="<?php echo $pluginSupportLink; ?>" target="_blank"><?php echo __("Get support"); ?> <span class="sr-only"><?php echo __('(external link)'); ?></span></a>`

to produce 

<img width="98" height="24" alt="Screenshot of link tagged with an 'external' icon" src="https://github.com/user-attachments/assets/d07a79bf-dfca-417d-8cf4-a05acda2e952" />

Screen readers announce "Get support (external link)". I'm open to changing the screen reader text. I hesitated to go with "Opens in X" because I'm not sure if we know if external links default to new tab or window. I think that's a browser setting.

**Comment by aviannamiller** [comment]
- This is announcing well across screen readers. Depending on the screen reader (such as with NVDA, TalkBack, VoiceOver/mobile), you sometimes get a bit of repetition with something like "Get support, external link, link", but I think we can compromise on the language here for reasons you have already stated.
- Missing from some View Public Page links, such as in Appearance > Navigation
- I've looked at a few instances of where external links might appear in plugin content. For Local Contexts, the external link icon appears admin-side, but it does not include the accessible text. The icon and text both do not carry over to the external links as they appear on sites.
- I'm noticing the icon picking up a font style where it maybe shouldn't, here on the theme selection page:
<img width="702" height="369" alt="Detail of theme for selection. Under 'Use this theme' button, external link icons appear next to two different links. The first link is italicized, and so is the icon, which appears a bit distorted in the process." src="https://github.com/user-attachments/assets/d6b216e3-51f2-40e1-951b-8f29ff807ac4" />

**Comment by kimisgold** [comment]
There should be fixes in response to all your points, except for Local Contexts. I think within the scope of this issue, we focus on the core, then file issues within the affected module repos.

**Comment by aviannamiller** [comment]
Sounds good to me. Changes look great on this, so I think we're covered here. I'll open an issue for Local Contexts.
